ansible/roles/test/files/ptftests/fib.py

```diff
@@ -58,22 +60,22 @@ def __init__(self, file_path):
             for line in f.readlines():
                 if pattern.match(line): continue
                 entry = line.split(' ', 1)
-                prefix = ip_network(unicode(entry[0]))
+                prefix = ip_network(unicode(entry[0]) if sys.version_info[0] == 2 else entry[0])
```

Comment:
Is there another better way to making unicode string for python2 and 3 ?

Reply:
According to Xin's suggestion, we can leverage the library `six` for portable text string at here.